### PR TITLE
fix(cluster): fix outdated doc in cluster

### PR DESCRIPTION
**Pull request by educhastenier**
Relates to https://github.com/bonitasoft/bonita-distrib-sp/pull/578

### CODE REVIEW

File: modules/runtime/pages/install-a-bonita-bpm-cluster.adoc [hunk 1]
```diff
@@ -49,11 +49,6 @@ When done you will have a database with all tables created and with a table `CON
 In the following steps, you will update the configuration files that are in the `setup/platform_conf/initial` folder of the platform setup tool.
 
 . Edit the file `setup/platform_conf/initial/platform_engine/bonita-platform-sp-custom.properties`: Uncomment and set the *`bonita.cluster`* property to `true`, as follows: `bonita.cluster=true`
-. Edit the file `setup/platform_conf/initial/platform_engine/bonita-platform-sp-custom.properties`: +++<a id="disable-hibernate-cache">++++++</a>+++In order to keep consistency between nodes, the Hibernate cache must be disabled:
-Uncomment and change the line:
-      `#bonita.platform.persistence.use_second_level_cache=true`
-    change it to:
-      `bonita.platform.persistence.use_second_level_cache=false`
```
Comment (JeremJR) [inline modules/runtime/pages/install-a-bonita-bpm-cluster.adoc:56]: shouldn't we also remove these lines https://github.com/bonitasoft/bonita-doc/pull/2434/files#diff-401c121520bcd01f543e593dd26eaace15a14c3798808a9fab4614d5c755d395R183-R186 ?

File: modules/runtime/pages/install-a-bonita-bpm-cluster.adoc [hunk 2]
```diff
@@ -68,6 +63,12 @@ bonita.platform.cluster.hazelcast.tcpip.members=ipServer01,ipServer02
 +
 IMPORTANT: If you don't use `bonita.platform.cluster.hazelcast.multicast.enabled`, *you must uncomment the # properties and set it to `false`* to deactivate it, as follows: `bonita.platform.cluster.hazelcast.multicast.enabled=false`.
 +
+
+[NOTE]
+====
+The https://www.google.com/search?q=Hibernate+L2+cache[Hibernate L2 cache] will be automatically disabled in cluster, leading to lower performance and more frequent database accesses.
```
Comment (JeremJR) [inline modules/runtime/pages/install-a-bonita-bpm-cluster.adoc]: Do we have details about that lower performance? isn't it a bit compensated in some way by adding more resources with new cluster nodes?

Comment (educhastenier) [reply modules/runtime/pages/install-a-bonita-bpm-cluster.adoc]: I'll reword it to avoid making it sound dramatic

**Review by JeremJR — APPROVED** [review]
LGTM
